code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -29,43 +118,16 @@ info:
     * __Edge Cloud__ - Cloud-like capabilities located at the network edge
     including, from the Application Provider's perspective, access to
     elastically allocated compute, data storage and network resources.
-    * __Edge Cloud Region__ - An OP Region is equivalent to a Region on a public
-    cloud. The higher construct in the hierarchy exposed to an Application
+    * __Edge Cloud Region__ - An Edge Cloud Provider Region is equivalent
+    to a Region on a Public Cloud.
+    The higher construct in the hierarchy exposed to an Application
     Provider who wishes to deploy an Application on the Edge Cloud and broadly
     represents a geography. A Region typically contains one or multiple Zones.
     A Region exists within an Edge Cloud.
     * __Edge Cloud Zone__ - An Edge Cloud Zone is the lowest level of
     abstraction exposed to an Application Provider who wants to deploy
     an Application on Edge Cloud. Zones exists within a Region
```

Comment:
"Zone exisits". I think we do not have an independent zone concept as written in this line. It may be Edge Cloud Zone that should be quoted here?